genesyscloud/resource_genesyscloud_telephony_providers_edges_phone.go

```diff
@@ -234,6 +246,37 @@ func readPhone(ctx context.Context, d *schema.ResourceData, meta interface{}) di
 	return nil
 }
 
+func assignUserToWebRtcPhone(ctx context.Context, sdkConfig *platformclientv2.Configuration, userId string) diag.Diagnostics {
+	stationsAPI := platformclientv2.NewStationsApiWithConfig(sdkConfig)
+	stationId := ""
+
+	retryErr := withRetries(ctx, 15*time.Second, func() *resource.RetryError {
+		stations, _, getErr := stationsAPI.GetStations(100, 1, "", "", "", userId, "", "")
```

Comment:
Why is the value of 100 set here.  Even if it is within a function we should not be using"magic numbers".  Instead use a constant to describe what the value is for.

Reply:
They are used for all calls to list resources (usually page number and size) so that change should be made across the board if we don't want magic numbers.